src/app/SecurityPanel/Credentials/StoreJmxCredentials.tsx

```diff
@@ -278,8 +252,6 @@ export const StoreJmxCredentials = () => {
 
 export const StoreJmxCredentialsCard: SecurityCard = {
   title: 'Store JMX Credentials',
-  description: `Targets for which Cryostat has stored JMX credentials are listed here.
-    If a Target JVM requires JMX authentication, Cryostat will use stored credentials
-    when attempting to open JMX connections to the target.`,
+  description: `Credentials that Cryostat uses to connect to target JVMs over JMX are stored here. Expand an item to see which known target applications the credential applies to.`,
```

Comment:
We cannot see the expanded yet right?

Reply:
Not yet, that's https://github.com/cryostatio/cryostat-web/issues/466 . I can alter the text here so that it isn't referencing that feature that isn't implemented just yet.